lib/galaxy/webapps/galaxy/api/wes.py

```diff
@@ -0,0 +1,162 @@
+"""GA4GH WES (Workflow Execution Service) API endpoints."""
+
+import logging
+from typing import (
+    Annotated,
+    Optional,
+)
+
+from fastapi import (
+    File,
+    Form,
+    Path,
+    Query,
+    Request,
+    UploadFile,
+)
+
+from galaxy.managers.context import ProvidesUserContext
+from galaxy.schema.fields import DecodedDatabaseIdField
+from galaxy.schema.wes import (
+    RunId,
+    RunListResponse,
+    RunLog,
+    RunStatus,
+    ServiceInfo,
+    TaskListResponse,
+    TaskLog,
+)
+from galaxy.webapps.galaxy.services.wes import WesService
+from galaxy.work.context import SessionRequestContext
+from . import (
+    depends,
+    DependsOnTrans,
+    Router,
+)
+
+log = logging.getLogger(__name__)
+router = Router(tags=["wes"])
+
+RunIdParam = Annotated[
+    DecodedDatabaseIdField,
+    Path(
+        title="Workflow Invocation ID",
+    ),
+]
+TaskIdParam = Annotated[
+    str,
+    Path(
+        title="Task ID",
+        description="Task identifier: step order_index, or order_index.job_index for collection mapping jobs",
```

Comment:
Can you add an example here, I understand it's either `0` or `0.1` but not sure I read this correctly.